Dockerfile

```diff
@@ -14,6 +14,7 @@ RUN INSTALL_PKGS=" \
 ENV GOPATH=$HOME/go
 
 ADD dist/entrypoint.sh /root/entrypoint.sh
+ADD . /root/ovsdb-mon
```

Comment:
hmm... I like that! Since we are adding '.', do we even need `ADD dist/entrypoint.sh /root/entrypoint.sh` ?
Is it better to just remove that line and update entrypoint to be `ENTRYPOINT /root/dist/entrypoint.sh`
Just a thought. Otherwise
/lgtm!

Reply:
I think having a script called entrypoint.sh in /root is a common useful practice. If something goes wrong you can quickly look at it to get some context. How about replacing the "ADD" with a "RUN cp"?

Reply:
Make sense. lgtm!